agent/ecscni/plugin.go

```diff
@@ -311,6 +329,24 @@ func (client *cniClient) createENINetworkConfig(cfg *Config) (string, *libcni.Ne
 	return defaultENIName, networkConfig, nil
 }
 
+func (client *cniClient) createBranchENINetworkConfig(cfg *Config) (string, *libcni.NetworkConfig, error) {
+	eniConf := BranchENIConfig{
+		TrunkMACAddress:          cfg.TrunkMACAddress,
+		Type:                     ECSBranchENIPluginName,
+		CNIVersion:               client.cniVersion,
+		BranchVlanID:             cfg.BranchVlanID,
+		BranchIPAddress:          cfg.ENIIPV4Address,
+		BranchMACAddress:         cfg.ENIMACAddress,
+		BlockInstanceMetdata:     cfg.BlockInstanceMetdata,
+		SubnetGatewayIPV4Address: cfg.SubnetGatewayIPV4Address,
```

Comment:
this is not a field for the new plugin. instead this should be BranchGatewayIPAddress